autowebcompat/network.py

```diff
@@ -10,6 +10,7 @@
     'nadam': Nadam(),
     'rms': RMSprop()
 }
+CLASSIFICATION_TYPE = ['Y vs N + D', 'Y + D vs N']
```

Comment:
`CLASSIFICATION_TYPE` doesn't belong in this module, as this module is only about neural network-related stuff.

Reply:
@marco-c do you have any name in mind for this

Reply:
The name is OK (maybe use the plural). It's the module where you put it that isn't OK.